src/autoscaler/db/sqldb/lock_sqldb_test.go

```diff
@@ -98,7 +98,7 @@ var _ = Describe("LockSqldb", func() {
 		Context("when the lock does not exist", func() {
 			Context("because the row does not exist", func() {
 				BeforeEach(func() {
-					lock = &models.Lock{Owner: "123456", Ttl: testTTL}
+					lock = createLock("123456", testTTL)
```

Comment:
I'm not sure these changes should be in here, they seem to have come from the other PR.

Reply:
yeah its a bit old Ill update.

Reply:
out of this PR :)